tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipseUiRunnerTest.java

```diff
@@ -34,7 +34,6 @@ public void emptyClassAndEmptyTest() throws CoreException {
         runTest(FOO_TEST_CLASS, FOO_BAR_PACKAGE, TEST_PROJECT);
         consoleContains(0, 0, 100, 0, 0);
         mutationsAre(Collections.emptyList());
-        coverageReportGenerated(0, 100, 100);
```

Comment:
this check has been removed

Reply:
This check internally refers to the created HTML page from Pit, which are not created anymore since the new update, if nothing is mutated.

Reply:
But Pitclipse will create an empty HTML will it not?

Reply:
No, the file is not created or at least not communicated with this extension point.